tyk-pro/scripts/init.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e
+
+apk --no-cache add curl jq
```

Comment:
It's good to add repo for each package as both packages are not available on main repository and v3.13 branch.

Reply:
I just moved what we had in the bootstrap. If you'd like to make the change, by all means.

Reply:
i think this can be in a different PR.

Reply:
Awesome thanks. I'll resolve this then.